feature-libs/checkout/base/core/facade/checkout-delivery-address.service.spec.ts

```diff
@@ -55,15 +55,23 @@ class MockCheckoutQueryFacade implements Partial<CheckoutQueryFacade> {
   );
 }
 
-// configure rxjs to not crash node instance with thrown errors
-config.onUnhandledError = (err) => console.warn(err);
-
 describe(`CheckoutDeliveryAddressService`, () => {
   let service: CheckoutDeliveryAddressService;
   let connector: CheckoutDeliveryAddressConnector;
   let checkoutQuery: CheckoutQueryFacade;
   let eventService: EventService;
 
+  beforeAll(() => {
+    // configure rxjs to not crash node instance with thrown errors
+    // TODO: CXSPA-4870 verify if can be avoided
+    config.onUnhandledError = () => {};
+  });
+
+  afterAll(() => {
+    // reset rxjs configuration
+    config.onUnhandledError = null;
+  });
```

Comment:
Shouldn't it be 
let originalRxjsOnUnhandledError;
```suggestion
  beforeAll(() => {
    // configure rxjs to not crash node instance with thrown errors
    // TODO: CXSPA-4870 verify if can be avoided
    originalRxjsOnUnhandledError = config.onUnhandledError;
    config.onUnhandledError = () => {};
  });

  afterAll(() => {
    // reset rxjs configuration
    config.onUnhandledError = originalRxjsOnUnhandledError;
  });
```